pkgs/applications/video/manim/failing_tests.nix

```diff
@@ -65,6 +65,8 @@
 # E           Max relative difference: 255.
 [
   "test_Text2Color"
+  "test_PointCloudDot"
+  "test_Torus"
```

Comment:
Just to be sure: are they failing for the same reason?

Reply:
The msgs of the failing tests are not exactly equal but very similar to the above. That's why I assume there is a common cause for the failures.

1) `test_PointCloudDot` fails with:
```
E           AssertionError:
E           Not equal to tolerance rtol=1e-07, atol=1.01
E           Frame no -1. You can use --show_diff to visually show the difference.
E           Mismatched elements: 8 / 1639680 (0.000488%)
E           Max absolute difference: 255
E           Max relative difference: 0.00787402
```

2) `test_Torus` fails with:
```
E           AssertionError:
E           Not equal to tolerance rtol=1e-07, atol=1.01
E           Frame no -1. You can use --show_diff to visually show the difference.
E           Mismatched elements: 1 / 1639680 (6.1e-05%)
E           Max absolute difference: 255
E           Max relative difference: 2.42857143
```